LayoutTests/fast/forms/color/color-type-change-on-input-crash.html

```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
 <html>
-<script src="../../../resources/js-test.js"></script>
-</head>
 <body>
+    <script src="../../../resources/js-test.js"></script>
```

Comment:
Nit: Remove indentation to match the line below.

Reply:
Done